docs/security/hardening-guide.md

```diff
@@ -464,6 +560,165 @@ sudo systemctl daemon-reload
 sudo systemctl restart k3s.service
 ```
 
+## Configuration for Kubernetes Components
+
+
+The configuration below should be placed in the [configuration file](../installation/configuration.md#configuration-file), and contains all the necessary remediations to harden the Kubernetes components.
+
+
+<Tabs>
+<TabItem value="v1.25 and Newer" default>
+
+```yaml
+protect-kernel-defaults: true
+secrets-encryption: true
+kube-apiserver-arg:
+  - 'enable-admission-plugins=NodeRestriction,NamespaceLifecycle,ServiceAccount'
```

Comment:
Aren't all three of these these enabled by default?

Reply:
GH review is somewhat weird, do you mean: secrets-encryption, protect-kernel-defaults, and the apiserver args as the 3? or the 3 admission plugins?

Reply:
The answer is no for both. The only default that is enabled for the plugins is `NodeRestriction` so that can be removed from this guide.

Reply:
Which you can't do because once you define the plugins as an argument, it overrides the k3s default (just tested). So all 3 must be included.

Reply:
According to the upstream docs:
> --enable-admission-plugins strings
> admission plugins that should be enabled in addition to default enabled ones (NamespaceLifecycle, LimitRanger, ServiceAccount, TaintNodesByCondition, PodSecurity, Priority, DefaultTolerationSeconds, DefaultStorageClass, StorageObjectInUseProtection, PersistentVolumeClaimResize, RuntimeClass, CertificateApproval, CertificateSigning, CertificateSubjectRestriction, DefaultIngressClass, MutatingAdmissionWebhook, ValidatingAdmissionPolicy, ValidatingAdmissionWebhook, ResourceQuota).

So these are **additional** plugins, beyond the defaults - and NamespaceLifecyle and ServiceAccount are already covered by the defaults, so they don't need to be included, either in the K3s defaults, or those set by the user.

Reply:
Doing some testing, I have removed these from the 1.25 config.